MovieDBApp/MovieDBApp/Platform/Services/MovieURLs.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MovieURLs.swift
+//  MovieDBApp
+//
+//  Created by Phong Le on 28/07/2021.
+//
+
+import Foundation
+
+class MovieURLs {
+    static private let sharedMovieURLs: MovieURLs = {
+        let movieURLs = MovieURLs()
+        return movieURLs
+    }()
+    
+    class func shared() -> MovieURLs {
+        return sharedMovieURLs
+    }
+    
+    private let baseURL = "https://api.themoviedb.org/3"
+    
+    private init() {}
```

Comment:
sắp xếp lại thứ tự fun và biến. Cái nào không dùng thì bỏ đi
```suggestion
    private let baseURL = "https://api.themoviedb.org/3"

    class func shared() -> MovieURLs {
        return sharedMovieURLs
    }
    
    
```